tools/export_program_info/export_program_info.cpp

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#define USER_MODE
+#define _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING
+#include <codecvt>
+
+#include "ebpf_api.h"
+#include "ebpf_nethooks.h"
+#include "ebpf_store_helper.h"
+#include "export_program_info.h"
+#include "windows_program_type.h"
+
+#include "ebpf_general_helpers.c"
+
+#define REG_CREATE_FLAGS (KEY_WRITE | DELETE | KEY_READ)
+#define REG_OPEN_FLAGS (DELETE | KEY_READ)
+
+// TODO: Issue #1231 Change to using HKEY_LOCAL_MACHINE
+ebpf_registry_key_t root_registry_key = HKEY_CURRENT_USER;
+
+#define SIZE_OF_ARRAY(x) (sizeof(x) / sizeof(x[0]))
```

Comment:
You don't need this, C++ already has _countof(x).
See https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/countof-macro?view=msvc-170